src/main/resources/themes/one_dark.theme.json

```diff
@@ -312,8 +312,7 @@
 
       "Header": {
         "background": "#414855",
-        "inactiveBackground": "#323844",
-        "borderColor": "#21252b"
```

Comment:
this borderColor setting made the header look like 2px smaller in height compared to the editor tabs

![image](https://user-images.githubusercontent.com/284129/80837229-3e447900-8c18-11ea-8002-d8753738ebef.png)